src/collectors/mongodb/test/testmongodb.py

```diff
@@ -6,13 +6,18 @@
 from test import get_collector_config
 from test import unittest
 from test import run_only
-from mock import MagicMock
-from mock import patch
-from mock import call
+from test import MagicMock
+from test import patch
+from test import call
 
 from diamond.collector import Collector
 from mongodb import MongoDBCollector
 
+try:
+    long
```

Comment:
fwiw, in python2, there's literally no reason to ever use `long` afaik. `int()` will automatically coerce to a `long` if the value is long enough. So we can just stop using `long()` entirely.